docs/components/cards.md

```diff
@@ -0,0 +1,478 @@
+                            What effect will shopping for
+                            an auto loan have on my credit?
+                            <a href="#" class="a_link">
+                                Read answer
+                            </a>
+                        </li>
+
+                        <li class="m-list_item">
+                            I owe more on my current loan than my current
+                            vehicle is worth. What do I need to know
+                            if I buy a new vehicle?
+                            <a href="#" class="a_link">
+                                Read answer
+                            </a>
+                        </li>
+
+                    </ul>
+
+                    <p class="m-card_footer">
+                        <a href="#">
+                            Learn more about bank accounts
+                        </a>
+                    </p>
+                </article>
+
+                <article class="m-card">
+                    <h2 class="m-card_heading">
+                        <a class="" href="#">
+                            <div class="m-card_icon">{% include icons/credit-card.svg %}</div>
+                            <span>Credit cards</span>
+                        </a>
+                    </h2>
+
+                    <ul class="m-list">
+
+                        <li class="m-list_item">
+                            What is a credit card interest rate? What does APR mean?
+                            <a href="#" class="a_link">
+                                Read answer
+                            </a>
+                        </li>
+
+                        <li class="m-list_item">
+                            How do I dispute a charge on my credit card bill?
+                            <a href="#" class="a_link">
+                                Read answer
+                            </a>
+                        </li>
+
+                    </ul>
+
+                    <p class="m-card_footer">
+                        <a href="#">
+                            Learn more about bank accounts
+                        </a>
+                    </p>
+                </article>
+            </div>
+        </div>
+
+  - variation_name: Featured Card
+    variation_description: |-
+        A single featured card.
+        The `m-card__featured` modifier is added.
+    variation_code_snippet: |-
+        <article class="m-card m-card__featured">
+            <a class="" href="#">
+                <h2 class="m-card_heading">
+                    <div class="m-card_icon">
+                        {% include icons/idea.svg %}
+                    </div>
+                </h2>
+                <p>
+                    Have a problem with a financial product or service?
+                </p>
+                <div class="m-card_footer">
+                    <span>Submit a complaint</span>
+                </div>
+            </a>
+        </article>
```

Comment:
Single featured cards look real wide and silly on desktop but 🤷‍♂ .

Reply:
Yeeeeaah